packages/apalis-redis/src/storage.rs

```diff
@@ -632,6 +632,7 @@ where
             }
             Err(e) => {
                 warn!("An error occurred during streaming jobs: {e}");
+                self.keep_alive(worker_id).await?;
```

Comment:
Different errors may occur while streaming, could you include a check? Something like:
```suggestion
           if e.to_string().contains("consumer not registered script") {
                self.keep_alive(worker_id).await?;
           }
```

Reply:
Even better if you use the actual redis error.